### PR TITLE
[hyd-826] extract binaries from plugins and add them to velero image

### PR DESCRIPTION
**Pull request by debianmaster**

### CODE REVIEW

**Review by dinesh — CHANGES_REQUESTED** [review]
I guess we should aim for a reproducible docker image for local also. Can we create a bash script/make targets to extract the binaries and build image?  If Github Actions is not available, building local image using docker CLI will fail.

**Comment by debianmaster** [comment]
@dinesh  understood, i will add scripts for local build

File: hack/cape-custom-scripts.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+echo $(pwd)
```
Comment (dinesh) [inline hack/cape-custom-scripts.sh]: prefer to add `set -eu` to stop the execution if any of the commands fail.

**Review by dinesh — COMMENTED** [review]
:+1: looks good. have a minor nit.

**Review by dinesh — APPROVED** [review]

File: hack/cape-custom-scripts.sh (hunk 1 above)
Comment (debianmaster) [reply hack/cape-custom-scripts.sh]: @dinesh  added

**Review by dinesh — APPROVED** [review]

**Pull request merged**